app/models/user.rb

```diff
@@ -110,9 +111,12 @@ def can_prefill_emis?
     beta_enabled?(uuid, FormProfile::EMIS_PREFILL_KEY)
   end
 
+  ID_CARD_ALLOWED_STATUSES = %w(V1 V3 V6).freeze
+
   def can_access_id_card?
-    beta_enabled?(uuid, 'veteran_id_card') && loa3? && edipi.present? && veteran?
-  rescue # Default to false for any veteran_status error
+    beta_enabled?(uuid, 'veteran_id_card') && loa3? && edipi.present? &&
```

Comment:
Performance NITPICK, possibly do the loa3? check before beta_enabled? check to avoid making a db call if not necessary. Maybe even edipi since MVI is going to get fetched one way or the other if loa3.